.github/workflows/ci.yml

```
@@ -19,7 +19,7 @@ jobs :
       # solution.
       fail-fast : false
       matrix :
-        kotlin-version : [ 1.7.20 ]
+        kotlin-version : [ 1.8.0-RC ]
```

```suggestion
        kotlin-version : [ 1.7.20, 1.8.0-RC ]
```
And everywhere else.

---

For sure! Wasn't intending these as the final changes, just wanted to shake out any 1.8 issues before going through the more final changes of supporting both at once (due to the new @ExperimentalCompilerAPI changes)